src/Artsy/Router/matchingMediaQueriesForUserAgent.ts

```diff
@@ -0,0 +1,31 @@
+import { findDevice } from "@artsy/detect-responsive-traits"
+import {
+  findBreakpointAtWidth,
+  findBreakpointsForWidths,
+  MatchingMediaQueries,
+} from "Utils/Responsive"
+
+/**
+ * Find the breakpoints and interactions that the server should render
+ */
+export function matchingMediaQueriesForUserAgent(
+  userAgent: string
+): MatchingMediaQueries {
+  const device = findDevice(userAgent)
+  if (!device) {
+    return undefined
+  } else {
+    const supportsHover = device.touch ? "notHover" : "hover"
```

Comment:
Not a blocker, but I wonder if there's a better way to encapsulate `hover` (and other media queries of that kind). Might be a bit premature tho.